### DISCUSSION TITLE
skywalking tomcat threadpool alarm not work

### DISCUSSION THREAD

**Discussion by liu729133700** [apache/skywalking#12423] [Q&A]
version 9.4.0

tomcat_thread_pool_active_alarm_rule:
    metrics-name: meter_thread_pool_active_size-tomcat_execute_pool
    op: ">"
    threshold: 1
    period: 1
    count: 1
    silence-period: 0
    message: instance {name} tomcat thread pool active size over 1.

This is one of my alarm configuration items.I used the jmeter tool to raise the instance active_size-tomcat_execute_pool indicator to 8, but the alarm did not work.I suspect that metrics-name is incorrectly configured

**Comment by wu-sheng** [comment]
Period 1 is not working. You should have a larger window.

**Comment by liu729133700** [comment]
![tomcat threadpool](https://github.com/apache/skywalking/assets/35722766/dad3f1e0-dfa4-41bd-bb5f-8503e7ab9b90)

    tomcat_thread_pool_active_alarm_rule:
    metrics-name: meter_thread_pool_active_size-tomcat_execute_pool
    op: ">"
    threshold: 2
    period: 5
    count: 1
    silence-period: 5
    message: 实例 {name} tomcat活跃线程数已超过 1.

I modified the period configuration entry but it still didn't trigger the alarm

**Reply by wu-sheng** [reply]
Isn't this  labeled metric? How your config works foe this?

I believe you should use 10.0.1 and learn mqe in altering.
